Enterprise-CMCS/macpro-onemac
# Add FAQ section and links from Attachments on form pages

Story: https://qmacbis.atlassian.net/browse/OY2-23750

### Details

This PR adds a new FAQ section about the various file types we allow for attachments. It also adds a link to that section along with some copy provided. Finally, it customizes the FAQ link that already exists to allow for routing to specific sections of the FAQ page (i.e. we link directly to that form's relevant attachment descriptions on the FAQ page). 

### Changes

- Upgrades to the components handling display of this UI (also enhances uniformity)
- New FAQ section added
- Updated ROUTES with form attachment related anchors

### Implementation Notes

N/A

### Test Plan

1. Using the New Submission button, navigate through the card options to a form
2. In the attachments section of that form, ensure the copy meets the requirements within the Jira ticket
3. Ensure the first FAQ link meets the requirement of navigating to the attachments FAQ section for the given form
4. Ensure the second FAQ link meets the requirement of navigating to the file types FAQ section
5. Repeat 1-4 with the form pages given in the Jira ticket's AC
